IdentityCore/src/MSIDError.m

```diff
@@ -29,6 +29,7 @@
 NSString *MSIDCorrelationIdKey = @"MSIDCorrelationIdKey";
 NSString *MSIDHTTPHeadersKey = @"MSIDHTTPHeadersKey";
 NSString *MSIDHTTPResponseCodeKey = @"MSIDHTTPResponseCodeKey";
+NSString *MSIDHTTPResponseStringKey = @"MSIDHTTPResponseStringKey";
```

Comment:
Let's call it, MSIDTruncatedHTTPResponseStringKey?

Reply:
addressed